mod/01_rating/spec/set/right/downvotee_search_spec.rb

```diff
@@ -121,7 +121,16 @@
               "+company"=>{"content"=>"[[#{company.name}]]",:type_id=>Card::PointerID},
               "+value"=>{"content"=>"Na einai kalytero anthropo apo ton patera toy", :type_id=>Card::PhraseID},
               "+year"=>{"content"=>"2015", :type_id=>Card::PointerID},
-              "+Link"=>{:content=>"http://www.google.com/?q=fringepeter", "type_id"=>Card::PhraseID}
+              "+source"=>{
+                "subcards"=>{
+                  "new source"=>{
+                    "+Link"=>{
+                      "content"=>"http://www.google.com/?q=fringepeter",
+                       "type_id"=>Card::PhraseID
+                    }
+                  }
+                }
+              }
```

Comment:
the new API was supposed to clean things up, but this PR adds a lot more than it removes.  And this now appears to skip the "+Link" handling that is still in the code, no?